src/strings.cpp

```diff
@@ -448,13 +448,8 @@ static char *FormatTinyOrISODate(char *buff, Date date, StringID str, const char
 	YearMonthDay ymd;
 	ConvertDateToYMD(date, &ymd);
 
-	char day[3];
-	char month[3];
 	/* We want to zero-pad the days and months */
```

Comment:
Isn't the comment now not a bit weird?

Reply:
Yes because there is no zero-padding to be seen in the function, and no because it answers the question: why the twos after the day and month? Though maybe the code needs to be more pedantically obvious, so the comment isn't needed?

Reply:
This is just an annoying thing of our string system honestly, hard to make sure people understand this in a year or 2 :P

Suggestion: `Day and month are zero-padded with ZEROFILL_NUM, hence the two 2s`

Or something like that.